### DISCUSSION TITLE
How to get amalgamated parquet extension after 0.7.0

### DISCUSSION THREAD

**Discussion by Yohahaha** [duckdb/duckdb#7026] [Q&A]
Hi all,

Velox use DuckDB to verify execution results, this [docs](https://github.com/facebookincubator/velox/blob/main/velox/duckdb/README.md) show a way to get amalgamated file. However, I see this [commit](https://github.com/duckdb/duckdb/commit/64597fae4b824de1d82727b1d11e8051bb503cae) remove parquet-amalgamation.py. I have no ideas about how to get amalgamated parquet extension in currnet 0.7.1, could someone tell me the newly way?

**Comment by Mytherin** [comment]
The Parquet amalgamation has been deprecated and removed - you could e.g. copy over the source files of the Parquet extension instead, or use a pre-built version of DuckDB with the Parquet library included.
